src/DashboardUI/src/pages/admin/AdminOrganizationUsersPage.tsx

```diff
@@ -61,23 +67,86 @@ export function AdminOrganizationsUsersPage() {
     }
   };
 
+  const openRemoveUserModal = (userId: string) => {
+    setShowRemoveUserModal(!showRemoveUserModal);
+    setRemoveUserId(userId);
+  };
+
+  const closeRemoveUserModal = () => {
+    setShowRemoveUserModal(false);
+    setRemoveUserId('');
+  };
+
   return (
     <>
       <div className="overflow-y-auto h-100">
         <div className="m-3">
           {pageTitle()}
 
-          <div className="d-flex justify-content-between my-4">
-            <h2>Users</h2>
+          <div className="d-flex justify-content-between align-items-center my-4">
+            <h2 className="fs-5">Users</h2>
             {addUserButton()}
           </div>
 
           <TableLoading
             isLoading={organizationListLoading || organizationUsersListLoading}
             isErrored={organizationListErrored || organizationUsersListErrored}
           >
-            (<pre>OrganizationUsers: {JSON.stringify(organizationUsersListResponse)}</pre>)
+            <table className="w-100">
+              <thead>
+                <tr>
+                  <th className="px-2 py-1 border-bottom">User</th>
+                  <th className="px-2 py-1 border-bottom">Role</th>
+                  <th className="px-2 py-1 border-bottom">Email</th>
+                  <th className="px-2 py-1 border-bottom">User ID</th>
+                  <th className="px-2 py-1 border-bottom"></th>
+                </tr>
+              </thead>
+              <tbody>
+                {organizationUsersListResponse?.users &&
+                  organizationUsersListResponse?.users.length > 0 &&
+                  organizationUsersListResponse?.users.map((user) => (
+                    <tr key={user.userId}>
+                      <td key={`${user.userId}-name`} className="px-2 py-1 border-bottom">
+                        {user.lastName}, {user.firstName}
```

Comment:
We have some places in the app that do first last, and some that do last, first. Let's go with first last

```suggestion
                        {user.firstName} {user.lastName}
```